src/index.js

```diff
@@ -141,7 +144,7 @@ export default class CollapsibleToolbar extends Component {
                         }
                     ]}
                 >
-                    {renderNavBar()}
+                    {renderNavBarNoTitle ? renderNavBarNoTitle() : renderContent()}
```

Comment:
Why do you want to render content in navbar ? Can you provide an use case for this ?

Reply:
In my case I wanted to render one component when the header was open and another when they were closed

Reply:
Sorry for the late reply. As i understand, this conditional function call can be done from app side. So it is not needed here, unless I'm mistaken about it's purpose.